crypto/ed25519/math/point.go

```diff
@@ -0,0 +1,92 @@
+package math
+
+import (
+	"crypto/subtle"
+
+	"chain/crypto/ed25519/internal/edwards25519"
+)
+
+type (
+	// Point is a point on the ed25519 curve.
+	Point edwards25519.ExtendedGroupElement
+
+	// EncodedPoint is a serialized representation of a point.
+	EncodedPoint [32]byte
+)
+
+// ZeroPoint is the zero point on the ed25519 curve.
+var ZeroPoint Point
```

Comment:
Could be as direct as

```
var ZeroPoint Point = zeroPoint()
```

as long as there's *some* indication it has a non-"zero" value.